platform/lib/platform_web/live/adminland_live/user_list_live.ex

```diff
@@ -60,15 +60,12 @@ defmodule PlatformWeb.AdminlandLive.UserListLive do
                             />
                           </div>
                           <div class="ml-4">
-                            <%= live_patch(
-                              to: "/profile/#{user.username}",
-                              class: "text-button group outline-none"
-                            ) do %>
+                          <.link patch="/profile/#{user.username}" class = "text-button group outline-none">
```

Comment:
The value of patch must be wrapped in `{}` if you want Elixir string interpolation.

Reply:
Also, make sure that your indentation is right (I think this should be one more level indented). You can fix that automatically by running `mix format`.